webdriverio/web/webdriver/package.json

```diff
@@ -6,7 +6,11 @@
   "scripts": {
     "test.local": "wdio test/configs/wdio.local.chrome.conf.js",
     "test.saucelabs.eu": "REGION=eu wdio test/configs/wdio.saucelabs.conf.js",
-    "test.saucelabs.us": "wdio test/configs/wdio.saucelabs.conf.js"
+    "test.saucelabs.us": "wdio test/configs/wdio.saucelabs.conf.js",
```

Comment:
If we are adding iOS and Android specific scripts here I would suggest to rename the other scripts to something like `test.saucelabs.desktop`, what do you think?

Reply:
done!